### ISSUE TITLE
Validation: It would be nice to validate examples against their JSON schema

### ISSUE THREAD

**Issue by EricWittmann** [Apicurio/apicurio-studio#995]
We currently don't validate the content of an example set on e.g. a Data Type.  It would be very cool if we could perform JSON Schema validation on the example.  Since we know the schema!

**Comment by adiohana** [comment]
@EricWittmann 
Are you referring to add a new capability for validating JSON object with data against its schema?
We are using Apicurio and it would be very helpful to consume this via API

**Comment by EricWittmann** [comment]
Yes - this issue refers to a new capability.   Currently we are validating the API Design while it's being edited.  And obviously displaying any errors/warnings we find.  One thing we are not doing is validating the **examples** against the JSON Schema.  For example with Data Types:

![image](https://user-images.githubusercontent.com/1890703/97175335-bd4ee000-1769-11eb-9e3b-477a9cbd4302.png)

In the screenshot above there is a Data Type called `RuleViolationCause` with two properties `context` and `description`.  This data type has an *Example* which is a little JSON snippet.  We are not currently checking that the JSON snippet (the example) is valid for the data type.   But we could!
